components/fieldlabel/index.css

```diff
@@ -10,47 +10,62 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  --spectrum-fieldlabel-bottom-to-text: var(--spectrum-component-bottom-to-text-75);
+  --spectrum-fieldlabel-font-size: var(--spectrum-font-size-75);
+  --spectrum-fieldlabel-line-height: var(--spectrum-line-height-200);
+  --spectrum-fieldlabel-asterisk-gap: var(--spectrum-field-label-top-to-asterisk-medium);
+  --spectrum-fieldlabel-side-padding-top: var(--spectrum-component-top-to-text-75);
+  --spectrum-fieldlabel-side-padding-right: var(--spectrum-spacing-200);
+  --spectrum-field-label-text-to-asterisk: var(--spectrum-field-label-text-to-asterisk-medium);
 }
-
 .spectrum-FieldLabel--sizeL {
-  @remapvars {
-    find: --spectrum-fieldlabel-l-;
-    replace: --spectrum-fieldlabel-;
-  }
+  --spectrum-fieldlabel-top-to-text: var(--spectrum-component-top-to-text-100);
+  --spectrum-fieldlabel-bottom-to-text: var(--spectrum-component-bottom-to-text-100);
+  --spectrum-fieldlabel-font-size: var(--spectrum-font-size-100);
+  --spectrum-fieldlabel-line-height: var(--spectrum-CJK-line-height-100);
+  --spectrum-fieldlabel-asterisk-gap: var(--spectrum-field-label-top-to-asterisk-large);
+  --spectrum-fieldlabel-side-padding-top: var(--spectrum-component-top-to-text-100);
+  --spectrum-fieldlabel-side-padding-right: var(--spectrum-spacing-200);
+  --spectrum-field-label-text-to-asterisk: var(--spectrum-field-label-text-to-asterisk-large);
 }
-
 .spectrum-FieldLabel--sizeXL {
-  @remapvars {
-    find: --spectrum-fieldlabel-xl-;
-    replace: --spectrum-fieldlabel-;
-  }
+  --spectrum-fieldlabel-top-to-text: var(--spectrum-component-top-to-text-200);
+  --spectrum-fieldlabel-bottom-to-text: var(--spectrum-component-bottom-to-text-200);
+  --spectrum-fieldlabel-font-size: var(--spectrum-font-size-200);
+  --spectrum-fieldlabel-line-height: var(--spectrum-CJK-line-height-200);
+  --spectrum-fieldlabel-asterisk-gap: var(--spectrum-field-label-top-to-asterisk-extra-large);
+  --spectrum-fieldlabel-side-padding-top: var(--spectrum-component-top-to-text-200);
+  --spectrum-fieldlabel-side-padding-right: var(--spectrum-spacing-200);
+  --spectrum-field-label-text-to-asterisk: var(--spectrum-field-label-text-to-asterisk-extra-large);
 }
 
 .spectrum-FieldLabel {
   display: block;
 
   box-sizing: border-box;
 
-  padding-block: var(--spectrum-fieldlabel-padding-top) var(--spectrum-fieldlabel-padding-bottom);
+  padding-block: var(--spectrum-fieldlabel-top-to-text) var(--spectrum-fieldlabel-bottom-to-text);
   padding-inline: 0;
 
-  font-size: var(--spectrum-fieldlabel-text-size);
-  font-weight: var(--spectrum-fieldlabel-text-font-weight);
-  line-height: var(--spectrum-fieldlabel-text-line-height);
+  font-size: var(--spectrum-fieldlabel-font-size);
+  font-weight: var(--spectrum-font-weight-regular);
+
+  /* line heights coming from core-tokens do not calculate correctly to the previous line-heights */
+  line-height: calc(var(--spectrum-fieldlabel-line-height) * var(--spectrum-fieldlabel-font-size));
```

Comment:
@pfulton some of those line-heights don't match exactly the previous line-heights.
Is this calc the correct way to get a line-height?

Reply:
Good question! I'll need to dig into this to get you a better answer.

Reply:
Sounds good, lets double check with design just to make sure this is intentional

Reply:
Yeah I'd avoid this `calc()` the values above should have the `calc`, if anything.

Reply:
That looks good to me! Is `CJK` as a prefix already an established convention? It seems odd to see capitalized letters, and I wonder if a suffix is more in line with what we have already. Your call.

Reply:
> That looks good to me! Is `CJK` as a prefix already an established convention? It seems odd to see capitalized letters, and I wonder if a suffix is more in line with what we have already. Your call.

I think so - it's in the core tokens file. 
I think Jess would be the one that decides on the final token names.